games-util/gtkevemon/gtkevemon-1.10.ebuild

```diff
@@ -1,13 +1,13 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
 inherit flag-o-matic
 
 DESCRIPTION="A standalone skill monitoring application for EVE Online"
-HOMEPAGE="http://gtkevemon.battleclinic.com"
-SRC_URI="http://github.com/gtkevemon/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="https://github.com/gtkevemon/gtkevemon"
```

Comment:
Agreed, I can change this commit upon rebasing the branch and resubmitting. I forgot to mention that HOMEPAGE was defunct in the commit message.